OloEngine/src/OloEngine/Renderer/MeshOptimization.cpp

```
@@ -0,0 +1,647 @@
+#include "OloEnginePCH.h"
+#include "OloEngine/Renderer/MeshOptimization.h"
+#include "OloEngine/Renderer/MeshSource.h"
+#include "OloEngine/Renderer/Mesh.h"
+#include "OloEngine/Renderer/Vertex.h"
+#include "OloEngine/Asset/AssetManager.h"
+#include "OloEngine/Animation/MorphTargets/MorphTarget.h"
+#include "OloEngine/Animation/MorphTargets/MorphTargetSet.h"
+
+#include <meshoptimizer.h>
+
+#include <cmath>
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)